vllm/multimodal/utils.py

```diff
@@ -455,6 +457,209 @@ def run_dp_sharded_vision_model(image_input: torch.Tensor,
     return vision_embeddings
 
 
+def get_load_balance_assignment(sizes: list[int], num_gpus: int = 2):
+    """
+    Generate load balancing assignment and metadata 
+    for distributing data across GPUs.
+    The load is determined by the total image sizes,
+    not the number of images.
+    
+    Args:
+        sizes: The size of each image
+        num_gpus: Number of GPUs to balance across
+    
+    Returns:
+        shuffle_indices: 
+            Indices to reorder data for balanced loading
+        gpu_sample_counts: 
+            Number of samples assigned to each GPU
+        grouped_sizes_per_gpu: 
+            Total size assigned to each GPU
+    
+    Example:
+    
+        sizes = [1000, 100, 200, 50]
+        num_gpus=2
+
```

Comment:
```suggestion
    Example:
    
        ```
        sizes = [1000, 100, 200, 50]
        num_gpus=2
        ```
```